src/Components/AddItem.jsx

```diff
@@ -1,5 +1,85 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useForm } from 'react-hook-form';
+import firebase from 'firebase';
+import { db } from '../lib/firebase';
 
 export default function AddItem() {
-  return <div>Hi from Add an Item View!</div>;
+  const [token, setToken] = useState('test-token');
+  const { register, handleSubmit, reset, errors } = useForm();
+  const onSubmit = (data) => {
+    const currentList = db.collection('lists').doc(token);
```

Comment:
I like that you're thinking ahead here and saving the lists in terms of their token! 👍